scripts/main.js

```diff
@@ -0,0 +1,241 @@
+  return num[8];
+}
+
+const btnnine = document.getElementById('nine');
+btnnine.addEventListener('click', nine);
+
+function nine() {
+  disChar.innerHTML += num[9];
+  return num[9];
+}
+
+const btnfour = document.getElementById('four');
+btnfour.addEventListener('click', four);
+
+function four() {
+  disChar.innerHTML += num[4];
+  return num[4];
+}
+
+const btnfive = document.getElementById('five');
+btnfive.addEventListener('click', five);
+
+function five() {
+  disChar.innerHTML += num[5];
+  return num[5];
+}
+
+const btnsix = document.getElementById('six');
+btnsix.addEventListener('click', six);
+
+function six() {
+  disChar.innerHTML += num[6];
+  return num[6];
+}
+
+const btnone = document.getElementById('one');
+btnone.addEventListener('click', one);
+
+function one() {
+  disChar.innerHTML += num[1];
+  return num[1];
+}
+
+const btntwo = document.getElementById('two');
+btntwo.addEventListener('click', two);
+
+function two() {
+  disChar.innerHTML += num[2];
+  return num[2];
+}
+
+const btnthree = document.getElementById('three');
+btnthree.addEventListener('click', three);
+
+function three() {
+  disChar.innerHTML += num[3];
+  return num[3];
+  console.log(num[3])
+}
+
+const btnzero = document.getElementById('zero');
+btnzero.addEventListener('click', zero);
+
+function zero() {
+  disChar.innerHTML += num[0];
+  return num[0];
+}
+
+const operator = ['+', '/', '*', "-", '.']
+
+const btnmul = document.getElementById('mul');
+btnmul.addEventListener('click', mul)
+
+function mul () {
+  if (disResult.textContent !== ' ') {
+    let newCalc = disResult.textContent;
+    disChar.innerHTML = ' ';
+    disChar.innerHTML += newCalc + operator[2];
+    disResult.textContent = ' ';
+    
+  }else {
+    disChar.innerHTML += operator[2];
+  }
+  
+}
+
+const btnadd = document.getElementById('add');
+btnadd.addEventListener('click', add);
+
+function add () {
+  if (disResult.textContent === ' ') {
+    disChar.innerHTML += operator [0];
+    // let newCalc = disResult.textContent;
+    // disChar.innerHTML = ' ';
+    // disChar.innerHTML += newCalc + operator[0];
+    // disResult.textContent = ' ';
+  } else {
+    // disChar.innerHTML += operator[0];
+    let newCalc = disResult.textContent;
+    disChar.innerHTML = ' ';
+    disChar.innerHTML += newCalc + operator[0];
+    disResult.textContent = ' ';
+  }
+  // disChar.innerHTML += operator[0];
+  // return operator[0];
+}
+
+const btnsub = document.getElementById('sub');
+btnsub.addEventListener('click', sub);
+
+function sub () {
+  if (disResult.textContent !== ' ') {
+    const newCalc = disResult.textContent;
+    // disChar.innerHTML = ' ';
+    disChar.innerHTML = newCalc + operator[3];
+    disResult.textContent = ' ';
+    
+  }else {
+    disChar.innerHTML += operator[3];
+  }
+  // disChar.innerHTML += operator[3];
+  // return operator[3];
+}
+
+const btndivi = document.getElementById('divi');
+btndivi.addEventListener('click', divi);
```

Comment:
There's a need for you to be more descriptive with your naming conventions. Using names like `multi`, `divi`, etc is not descriptive enough. This may be okay for now, but in the future, it might not make sense to you or to any other person who checks your codebase.

Reply:
There's nothing wrong with using their full name as variable names.